web/packages/teleterm/src/ui/StatusBar/StatusBar.tsx

```diff
@@ -30,6 +30,8 @@ import { AccessRequestCheckoutButton } from './AccessRequestCheckoutButton';
 import { ShareFeedback } from './ShareFeedback';
 import { useActiveDocumentClusterBreadcrumbs } from './useActiveDocumentClusterBreadcrumbs';
 
+export const statusBarHeight = 28;
```

Comment:
Could you move this export to `web/packages/teleterm/src/ui/StatusBar/constants.ts`? I hate that this is a thing we should do, but there's no way around this. See https://github.com/vitejs/vite-plugin-react/tree/main/packages/plugin-react-swc#consistent-components-exports.

I think at some point we should enable that ESLint rule as it's easy to miss this.

Reply:
that's good to know, darn... i'm 100% sure i did this in other places...